### PR TITLE
[TEVA-2742] Show statistics tab on vacancy page

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-2742

## Screenshots of UI changes:
Screenshot 2021-07-30 at 10 53 33

### CODE REVIEW

File: config/locales/publishers.yml [hunk 1]
```diff
@@ -280,6 +281,19 @@ en:
         view_jobs: View your job listings
         what_happens_next: What happens next
       statistics:
+        show:
+          application_data: Application data
+          explanation: This is to ensure the anonymity of applications who submit equal opportunities information.
+          intro: This may include data from candidates who have withdrawn their applications.
+          listing_data: Listing data
+          rejected_applications: Rejected applications
+          saves_by_jobseeker: Saves by jobseeker
```
Comment (josephhull676) [inline config/locales/publishers.yml:290]: jobseekers*

File: config/locales/publishers.yml [hunk 2]
```diff
@@ -280,6 +281,19 @@ en:
...
+          shortlisted_applications: Shortlisted applications
+          title: Statistics
+          total_applications: Total applications
+          unread_applications: Unread applications
+          views_by_jobseeker: Views by jobseeker
```
Comment (josephhull676) [inline config/locales/publishers.yml:295]: jobseekers*

File: spec/system/publishers_can_view_vacancy_statistics_spec.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+RSpec.describe "Publishers can view a job application" do
+  let(:publisher) { create(:publisher) }
+  let(:organisation) { create(:school) }
+  let(:vacancy) { create(:vacancy, organisation_vacancies_attributes: [{ organisation: organisation }]) }
+
+  before do
+    create(:saved_job, vacancy: vacancy)
+    create(:job_application, :status_submitted, vacancy: vacancy)
+    create(:job_application, :status_reviewed, vacancy: vacancy)
```
Comment (josephhull676) [inline spec/system/publishers_can_view_vacancy_statistics_spec.rb:11]: Do we need to create a reviewed job application?

**Review by josephhull676 — APPROVED** [review]
Approved with comments :)

File: app/views/publishers/vacancies/job_applications/_banner.html.slim [hunk 4]
```diff
@@ -42,9 +42,10 @@
...
 
-- if vacancy.enable_job_applications? && vacancy.published? && !vacancy.pending?
-  .govuk-grid-row
-    .govuk-grid-column-full
-      = render(DashboardComponent.new) do |dashboard|
-        - dashboard.navigation_item item: (link_to t("buttons.manage_listing"), organisation_job_path(vacancy.id), class: "dashboard-component-navigation__link", "aria-current": ("page" if controller_name == "vacancies"))
+.govuk-grid-row
+  .govuk-grid-column-full
+    = render(DashboardComponent.new) do |dashboard|
+      - dashboard.navigation_item item: (link_to t("buttons.manage_listing"), organisation_job_path(vacancy.id), class: "dashboard-component-navigation__link", "aria-current": ("page" if controller_name == "vacancies"))
```
Comment (alexbowen) [inline app/views/publishers/vacancies/job_applications/_banner.html.slim:48]: not strictly part of this task but can you make these `govuk_link_to` (and below) so they get correct focus styles. should have done this in my my PR tbh.

**Review by alexbowen — APPROVED** [review]
apart from minor comment looks good

**Review by jesseyuen — APPROVED** [review]

**Pull request merged**